tensorflow/lite/micro/kernels/kernel_runner.cc

```diff
@@ -92,4 +98,4 @@ TfLiteStatus KernelRunner::Invoke() {
 }
 
 }  // namespace micro
-}  // namespace tflite
+}  // namespace tflite
```

Comment:
Is this change intentional/necessary?

Reply:
wait I'm confused which change line 101 ?